src/fvtt/page-script.js

```diff
@@ -37,7 +46,7 @@ class FVTTDisplayer {
             if (rollMode == "blindroll")
                 data["blind"] = true;
         } else {
-            data['type'] = MESSAGE_TYPES.OOC;
+            data['type'] = MESSAGE_TYPES.ROLL;
```

Comment:
How about whispers a couple of lines above it ?

Reply:
Changed.